app/src/main/admin/components/Users/SingleUser/ViewUserDetailsPage.tsx

```diff
@@ -6,24 +6,38 @@ import {UserTitle, UserTitleProps} from "./UserTitle";
 import {UserDetailsContent} from "./UserDetailsContent";
 import {userActions} from "../../../actions/UserActions";
 import {userStore} from "../../../stores/UserStore";
+import {IPageWithParent} from "../../../../shared/models/Breadcrumb";
+import {ViewAllUsersPage} from "../List/ViewAllUsersPage";
 
 export interface UserDetailsPageProps {
     username: string;
 }
 
 export class ViewUserDetailsPage extends AdminPageWithHeader<UserDetailsPageProps> {
-    componentDidMount() {
-        setTimeout(() => {
-            userStore.fetchUsers().catch(doNothing).then(() => {
-                userActions.setCurrentUser(this.props.location.params.username);
-            });
+    load() {
+        super.load();
+        userStore.fetchUsers().catch(doNothing).then(() => {
+            userActions.setCurrentUser(this.props.location.params.username);
         });
     }
 
+    name(): string {
+        return "User details";
```

Comment:
More useful to have the user's name here

Reply:
This is actually not easy to do, due to the order in which data arrives. I'd like to keep this PR/ticket from getting any more complicated, so I'll raise another ticket for this.